modules/ref-dashboard-configuration-options.adoc

```diff
@@ -55,6 +49,8 @@ disableProjects` | `false` | Shows the *Data Science Projects* option in the das
 | `dashboardConfig:
 disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
 | `dashboardConfig:
+disableServingRuntimeParams` | `false` | Shows the *Configuration parameters* section when deploying a model, or editing a deployed model, on the single-model serving platform. To hide this section, set the value to `true`.
```

Comment:
This part LGTM. Thanks @bredamc

Reply:
Thanks @manaswinidas! I added the dialog names for clarity, as you suggested - let me know what you think!